.github/workflows/pylint-checks.yml

```diff
@@ -58,6 +58,8 @@ jobs:
       run: |
         pip install -r requirements/pip.txt
         pip install -r requirements/edx/development.txt --src ${{ runner.temp }}
+        pip uninstall -y mysqlclient
+        pip install --no-binary mysqlclient mysqlclient
```

Comment:
We're having trouble today with mysqlclient, and I'm wondering if these lines are related. I don't see any commit message or comment explaining them. Any chance you remember?

Reply:
That change was copied in this PR from https://github.com/openedx/edx-platform/pull/28935. We did that because Mysql 8 was coming out-of-the-box for Ubuntu 18 so we installed and re-installed it to fix that issue.